hack/windows/Install-OVS.ps1

```diff
@@ -187,33 +128,272 @@ function InstallOVS() {
             }
         }
     }
+}
 
-    # Install OVS kernel driver.
-    Log "Installing OVS kernel driver"
-    $VMMSStatus = $(Get-Service vmms -ErrorAction SilentlyContinue).Status
-    if (!$VMMSStatus) {
-        $VMMSStatus = "not exist"
+function CheckAndInstallVCRedists {
+    param (
+        [Parameter(Mandatory = $true)] [String]$VCRedistPath,
+        [Parameter(Mandatory = $true)] [String]$VCRedistsVersion
+    )
+    $mininalVersion = [version]$VCRedistsVersion
+    $existingVCRedists = getInstalledVcRedists
+    foreach ($redist in $existingVCRedists) {
+        $installedVersion = [version]$redist.Version
+        # VC redists files with a higher version are installed, return.
+        if ($installedVersion -ge $mininalVersion) {
+            return
+        }
     }
-    Log "Hyper-V Virtual Machine Management service status: $VMMSStatus"
-    if ($VMMSStatus -eq "Running") {
-        cmd /c "cd $OVSDriverDir && install.cmd"
-    } else {
-        cd $OVSDriverDir ; netcfg -l .\ovsext.inf -c s -i OVSExt; cd $WorkDir
+    if (-not (Test-Path $VCRedistPath)) {
+        mkdir -p $VCRedistPath
+        curl.exe -LO $DefaultVCRedistsDownloadURL
```

Comment:
This doesn't look right: you are creating the `VCRedistPath` folder, but then you are downloading the `vc_redist.x64.exe` in the current working directory. Should it be something like `curl.exe -L -o $VCRedistPath/vc_redist.x64.exe`? Can you check it if works as expected?

Reply:
My bad, updated as `curl.exe -Lo $VCRedistPath\vc_redist.x64.exe $DefaultVCRedistsDownloadURL`.